cores/esp8266/LwipIntfDev.h

```diff
@@ -36,7 +36,7 @@ class LwipIntfDev: public LwipIntf, public RawDev
         _mtu(DEFAULT_MTU),
         _intrPin(intr),
         _started(false),
-        _default(false)
+        _default(true)
```

Comment:
was this intended?

Reply:
makes sense for one instance, true. ok if not a problem
(but, I wonder if default=false would be safer when there are more)

Reply:
If there are more than one interface, only the last "defaulted" one becomes the default route.
I will investigate further on why this is necessary, but last time I checked packets are not routed without it.
I could have delegated the directive to "userland" but I think it is better to let the library deal with defaults so we don't need to change the API or the guidelines / examples when something changes or is fixed.

Reply:
see also [WiFi](https://github.com/d-a-v/esp82xx-nonos-linklayer/blob/5919b910580265252ac1980b9fe936ece5839e41/glue-lwip/lwip-git.c#L326)

Reply:
so the instance variable does not make sense and it should be a static (and shared between the wifi and the lwipintf class), if every interface wants to become the default on callback?

Reply:
Fair enough.
I have moved the default value back to `false` and updated examples to explicitly use the new interface as default route.

Reply:
Should the lwip part also be made aware of default modifications? (with another issue / pr, I gather, not to pollute the changes too too much)

Reply:
Modification for "default" is reverted to what it was before the PR no ?
If the question is about "lwip2 - the driver adapter",  it is very delicate to update the WiFi interface management code per possible hidden side effect. Maybe a new gh-issue describing the API question might be indeed opened for that purpose.